### DISCUSSION TITLE
Synthesizing New 2D3D Pairs

### DISCUSSION THREAD

**Discussion by sunmengnan** [Nicholasli1995/EvoSkeleton#67] [Q&A] [answered]
大佬，你好：
我的应用场景多为摔倒，而HUMAN3.6M中没有这种场景，不知可否通过Synthesizing New 2D3D
Pairs中的Mutation Operator方法来针对性的生成这种场景数据？ 谢谢

**Comment by Nicholasli1995** [comment, marked as answer]
我觉得摔倒的场景姿态的变化可能不大,相比起来视角可能更重要. 你的应用场景应该有相机参数,建议你关注一下你的相机与人体相对的位置.然后训练网络的时候你就先用默认的数据,但是把H36M相机的焦距和外参(R, T)修改一下,尽量模拟你的实际场景.这样训练出来的模型应该能在你的场景表现好.

**Reply by sunmengnan** [reply]
好的， 相机的焦距和外参(R, T)都只能通过标定得到吗？这样还得去标定一下比较麻烦.....还有就是每个相机的相机的焦距和外参(R, T)都不同，如果像你说的用默认的Human3.6m数据，统一按照一个相机的焦距和外参来进行训练，是不是也造成了误差？还是说在这个场景下焦距和外参差别不大这种误差可以忽略？

**Reply by Nicholasli1995** [reply]
能标定是最好的,找一张棋盘格用个标定库很快就完成了.只需要内参就可以了.外参你可以估计人体的位置自己设.用同一个相机的参数训练和测试效果最好.
